apps/provisioning_api/lib/Controller/UsersController.php

```diff
@@ -167,19 +167,21 @@ public function getUsers(string $search = '', ?int $limit = null, int $offset =
 	 * @param string $search Text to search for
 	 * @param int|null $limit Limit the amount of groups returned
 	 * @param int $offset Offset for searching for groups
+	 * @param string $orderBy Field to order the results with
+	 * @param string $sort ASC or DESC
 	 * @return DataResponse<Http::STATUS_OK, array{users: array<string, Provisioning_APIUserDetails|array{id: string}>}, array{}>
 	 *
 	 * 200: Users details returned
 	 */
-	public function getUsersDetails(string $search = '', ?int $limit = null, int $offset = 0): DataResponse {
+	public function getUsersDetails(string $search = '', ?int $limit = null, int $offset = 0, string $orderBy = 'lastLogin', string $sort = 'DESC'): DataResponse {
```

Comment:
```suggestion
	public function getUsersDetails(string $search = '', ?int $limit = null, int $offset = 0, string $orderBy = 'uid', string $sort = 'asc'): DataResponse {
```
Same to not break the sort order + lowercase sort param